lib/ansible/module_utils/network/meraki/meraki.py

```diff
@@ -184,16 +184,30 @@ def get_org_id(self, org_name):
                     # self.fail_json(msg=i['id'])
                     return str(i['id'])
 
+    def get_nets(self, org_name=None, org_id=None):
+        if org_name:
+            org_id = self.get_org_id(org_name)
+        path = self.construct_path('get_all', org_id=org_id)
+        r = self.request(path, method='GET')
+        return json.loads(r)
+
     def get_net(self, org_name, net_name, data=None):
         ''' Return network information '''
         if not data:
             org_id = self.get_org_id(org_name)
-            path = '/organizations/{org_id}/networks/{net_id}'.format(org_id=org_id, net_id=self.get_net_id(org_name=org_name, net_name=net_name, data=data))
+            path = '/organizations/{org_id}/networks/{net_id}'.format(
```

Comment:
I think this would be more readable if you do:

```python
    org_id = self.get_org_id(org_name)
    net_id = self.get_net_id(org_name, net_name, data)
    path = '/organizations/{org_id}/networks/{net_id}'.format(org_id=org_id, net_id=net_id)
```

So using positional arguments where there's no discussion, moving different operations in separate steps, and bring the information together at the very end. The path-string could be coming from somewhere else (if this is something that is being reused, but probably not relevant here).

Reply:
No, no, the module name should definitely be singular.

I would probably use internally the same name as the REST API, especially because it is not exposed to the user and the internal stuff is in itself more consistent. But again, a personal choice, not something that is written into stone.

(I think the plurar in the REST interface does make sense and is a common thing even, because it does e.q. affect networks, but that is debatable).